src/main/java/io/vertx/json/schema/impl/SchemaValidatorImpl.java

```diff
@@ -928,12 +928,23 @@ private OutputUnit validate(final Object _instance, final JsonSchema schema, fin
         if (schema.containsKey("pattern") && !Pattern.compile(schema.get("pattern")).matcher((String) instance).find()) {
           errors.add(new OutputUnit(instanceLocation, computeAbsoluteKeywordLocation(schema, schemaLocation + "/pattern"), baseLocation + "/pattern", "String does not match pattern", OutputErrorType.INVALID_VALUE));
         }
-        if (
-          schema.containsKey("format") &&
-            !Format.fastFormat(schema.get("format"), (String) instance)
-        ) {
+        if (schema.containsKey("format") &&
```

Comment:
Isn't format only a annotation since 2020-12?

Reply:
Yes but before we weren't checking the draft. I think in a separate change we can go through and fix some of these keywords that only exist in one draft or another

Reply:
Okay you are right, this PR is only about contentEncoding